scripts/check_compliance.py

```diff
@@ -513,7 +513,9 @@ def run(self):
         scripts_path = os.path.join(ZEPHYR_BASE, "scripts", "dts")
 
         sys.path.insert(0, scripts_path)
+        logger.info("cd %s && ./testdtlib.py" % scripts_path)
```

Comment:
Think you could make this

```python
logger.info("cd %s && ./testedtlib.py", scripts_path)
```

The `logger` module has some built-in string formatting stuff.